packages/turf-tin/index.js

```diff
@@ -39,24 +39,42 @@ var featurecollection = helpers.featureCollection;
  */
 module.exports = function (points, z) {
     //break down points
```

Comment:
We might want to add some simple validation to test the input is correctly a FeatureCollection.

```javascript
if (points.type !== 'FeatureCollection') throw new Error('points must be a FeatureCollection');
```